package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "space-travelers",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.1",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
- [ ] As it is a requirement for this project and best practice, ensuring that the system can be validated for integration and data integrity, please kindly add `tests with React Testing Library and Jest (for unit tests)`. Please see screenshots of requirements below. 

![test1](https://user-images.githubusercontent.com/65852621/151350770-40a3a6ad-8e78-4e73-b427-c4baed853d08.png)

![test2](https://user-images.githubusercontent.com/65852621/151350838-0ede4d06-8e45-4340-866b-26fe691ccdcf.png)

Reply:
Sure 👍